docker/sysds.Dockerfile

```diff
@@ -60,14 +60,16 @@ RUN apt-get update -qq \
 	rm -r target/hadoop-test && \
 	rm -r target/maven-archiver && \
 	rm -r target/systemds-** && \
-	rm -r docker && \
+	# rm -r docker && \
 	rm -r docs && \
 	rm -r src && \
 	rm -r /usr/lib/mvn && \
 	rm -r CONTRIBUTING.md && \
 	rm -r pom.xml && \ 
 	rm -r ~/.m2
 
-COPY docker/mountFolder/main.dml /input/main.dml
+RUN mkdir /input && echo 'print("Hello from SystemDS")' > /input/main.dml
+
+# COPY docker/mountFolder/main.dml /input/main.dml
```

Comment:
i'm all for running it but don't remove the copy, so that the only change you make it  to add the RUN line.

Reply:
I am having error with running `COPY`

Reply:
This one

![image](https://user-images.githubusercontent.com/53068787/140474742-a2e52fbc-c569-4b98-9295-851d252697b9.png)

Reply:
this is happening because of your out commented line mentioned above.

Reply:
this one:

```
# rm -r docker && \
```